.github/workflows/get-streamers.yml

```diff
@@ -16,3 +15,20 @@ jobs:
         run: npm install
       - name: Run script
         run: npm run getstreamers
+      - name: Commit changes
+        run: |
+          something_changed=`git diff-index --exit-code --ignore-submodules HEAD`
+          if [ -n "$something_changed" ]
+          then
+              echo "Something changed, committing"
+              git config user.name github-actions
```

Comment:
Reference for the git commit part: https://github.com/actions/checkout#push-a-commit-using-the-built-in-token